packages/babel6/__tests__/babel6.test.js

```diff
@@ -1,15 +1,20 @@
 import test from 'ava'
+import sinon from 'sinon'
 import babel6 from '../index'
 
 test('Babel default options work', (t) => {
   const block = babel6()
 
+  const addLoader = sinon.spy((loaderDef) => config => config)
   const context = {
     fileType: () => '*.js'
   }
 
-  t.deepEqual(block(context), {})
-  t.deepEqual(block.post(context).module.loaders, [
+  t.deepEqual(block(context, { addLoader })({}), {})
+  t.deepEqual(block.post(context, { addLoader })({}), {})
```

Comment:
addLoader is a pure function, so it's usually a better idea to include/use the real one and do state based testing rather than setting it up as a spy. In general, I'd consider pushing towards doing snapshot testing for blocks, but in the meantime at least using the real utility functions would make these tests more valuable imo.

Also, I'm not entirely convinced that unit testing blocks is worthwhile. You get more bang for the buck by actually testing them in createConfig, I think.